AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests/HubEventTests/AuthHubEventHandlerTests.swift

```diff
@@ -0,0 +1,136 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+@testable import Amplify
+@testable import AWSCognitoAuthPlugin
+
+class AuthHubEventHandlerTests: XCTestCase {
+
+    var authHandler: AuthHubEventHandler!
+    override func setUp() {
+        try? Amplify.configure()
+        authHandler = AuthHubEventHandler()
+    }
+
+    override func tearDown() {
+        Amplify.reset()
+        authHandler = nil
+    }
+
+    /// Test whether HubEvent emits a signedIn event for normal signIn operation
+    ///
+    /// - Given: A listener to hub events
+    /// - When:
+    ///    - I mock a succesful sign in operation event
+    /// - Then:
+    ///    - I should receive a signedIn hub event
+    ///
+    func testSignedInHubEvent() {
+
+        let hubEventExpectation = expectation(description: "Should receive the hub event")
```

Comment:
This test does not do what it says, we are not doing a signIn in operation. Just mocking an event and listening to it.

Reply:
Fixed.. I just copied these tests from older tests..